substrate/utils/frame/omni-bencher/src/command.rs

```diff
@@ -0,0 +1,149 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::Parser;
+use frame_benchmarking_cli::BenchmarkCmd;
+use sc_cli::Result;
+use sp_runtime::traits::BlakeTwo256;
+
+/// # Polkadot Omni Benchmarking CLI
+///
+/// The Polkadot Omni benchmarker allows to benchmark the extrinsics of any Polkadot runtime. It is
+/// meant to replace the current manual integration of the `benchmark pallet` into every parachain
+/// node. This reduces duplicate code and makes maintenance for builders easier. The CLI is
+/// currently only able to benchmark extrinsics. In the future it is planned to extend this to some
+/// other areas.
+///
+/// General FRAME runtimes could also be used with this benchmarker, as long as they don't utilize
+/// any host functions that are not part of the Polkadot host specification.
+///
+/// ## Installation
+///
+/// Directly via crates.io:
+///
+/// ```sh
+/// cargo install --locked polkadot-omni-bencher
+/// ```
+///
+/// or when the sources are locally checked out:
+///
+/// ```sh
+/// cargo install --locked --path substrate/utils/frame/omni-bencher --profile=production
+/// ```
+///
+/// Check the installed version and print the docs:
+///
+/// ```sh
+/// polkadot-omni-bencher --help
+/// ```
+///
+/// ## Usage
+///
+/// First we need to ensure that there is a runtime available. As example we will build the Westend
+/// runtime:
+///
+/// ```sh
+/// cargo build -p westend-runtime --profile production --features runtime-benchmarks
```

Comment:
So, I've kinda experimented with how we can demonstrate CLI stuff in docs in a way that it doesn't break, and one way is this https://github.com/paritytech/polkadot-sdk/pull/3946/files#diff-0467122bb049905a5f904e93c5133c4a9648204b5a1d7f263fc5ad07d2154ffaR49

Wonder what you think and if it is worthwhile. 

We could have a docify post-processor someday that replaces a `Command` instance with its shell representation. Maybe its `Debug` or `Display` impl already does that :))

Reply:
Oh wow, it actually does it: https://doc.rust-lang.org/std/process/struct.Command.html#impl-Debug-for-Command